File: android/src/main/java/com/one_of_many_simons/one_button_game/MainActivity.java
package com.one_of_many_simons.one_button_game;

import android.os.Bundle;
import android.widget.FrameLayout;
import androidx.appcompat.app.AppCompatActivity;
import com.common.Data;
import com.common.Launcher;
import com.one_of_many_simons.one_button_game.graphics.Graphics;
import com.one_of_many_simons.one_button_game.utils.FileHandle;
import com.one_of_many_simons.one_button_game.utils.SystemUtils;

public class MainActivity extends AppCompatActivity {

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);

		FrameLayout frameLayout = new FrameLayout(this);
		setContentView(frameLayout);

		Data.graphics = new Graphics(this, frameLayout);
		Data.fileHandle = new FileHandle(this);
		Data.systemUtils = new SystemUtils(this);

		Launcher main = new Launcher();

		main.init();

//		this.finishAffinity();
	}
}

File: composeApp/src/androidMain/kotlin/com/one_of_many_simons/one_button_game/MainActivity.kt
package com.one_of_many_simons.one_button_game

import android.os.Bundle
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import com.one_of_many_simons.one_button_game.Libraries.fileHandle
import com.one_of_many_simons.one_button_game.utils.FileHandle

class MainActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        fileHandle = FileHandle()
        fileHandle.setContext(this)

        setContent {
            app()
        }
    }
}